src/core/include/openvino/core/node_input.hpp

```diff
@@ -92,7 +92,7 @@ class OPENVINO_API Input<const Node> {
     /// \return The element type of the input referred to by this input handle.
     OV_NO_DANGLING const element::Type& get_element_type() const;
     /// \return The shape of the input referred to by this input handle.
-    OV_NO_DANGLING const Shape& get_shape() const;
+    OV_NO_DANGLING Shape get_shape() const;
```

Comment:
```suggestion
    Shape get_shape() const;
```
Is `OV_NO_DANGLING ` required when return shape by value (copy)?